src/EnergyPlus/UnitHeater.cc

```diff
@@ -169,6 +169,8 @@ namespace UnitHeater {
 	Real64 QZnReq; // heating or cooling needed by zone [watts]
 	Array1D_bool MySizeFlag;
 	Array1D_bool CheckEquipName;
+	bool MyOneTimeFlag( true );
```

Comment:
@Nigusse Similar comments as in #5472 - add routinename to flag names, put in anonymous namespace, no extern